core/src/err.rs

```diff
@@ -23,6 +35,9 @@ impl Error {
                 GENERAL_ERROR
             }
             Error::MongoParseConnectionStringError(_) => UNABLE_TO_CONNECT,
+            Error::NoDatabase => NO_DSN_OR_DRIVER,
+            Error::ColIndexOutOfBounds(_) => COLUMN_NOT_FOUND,
```

Comment:
Take a look at this error [here](https://learn.microsoft.com/en-us/sql/odbc/reference/syntax/sqlgetdata-function?view=sql-server-ver16): 07009 Invalid descriptor index.  This part of the description sounds like a better fit: `The value specified for the argument Col_or_Param_Num was greater than the number of columns in the result set.`

Reply:
Good find, thanks for pointing this out